bundle/config/mutator/load_git_details.go

```diff
@@ -52,5 +53,16 @@ func (m *loadGitDetails) Apply(ctx context.Context, b *bundle.Bundle) error {
 		remoteUrl := repo.OriginUrl()
 		b.Config.Bundle.Git.OriginURL = remoteUrl
 	}
+
+	// Compute relative path of the bundle root from the Git repo root.
+	absBundlePath, err := filepath.Abs(b.Config.Path)
+	if err != nil {
+		return err
+	}
+	relBundlePath, err := filepath.Rel(repo.Root(), absBundlePath)
```

Comment:
Please add a comment stating that `repo.Root()` is always absolute.

Reply:
done